--- scripts/common	2019-01-25 15:09:48.121181010 +0100
+++ scripts/common	2019-01-25 15:11:11.100695017 +0100
@@ -93,9 +93,7 @@
 export -f on_chroot
 
 update_issue() {
-	local GIT_HASH
-	GIT_HASH=$(git rev-parse HEAD)
-	echo -e "Raspberry Pi reference ${IMG_DATE}\nGenerated using pi-gen, https://github.com/RPi-Distro/pi-gen, ${GIT_HASH}, ${1}" > "${ROOTFS_DIR}/etc/rpi-issue"
+	echo -e "DJO Infobord" > "${ROOTFS_DIR}/etc/rpi-issue"
 }
 export -f update_issue
 
