--- Dockerfile
+++ Dockerfile
@@ -1,4 +1,4 @@
-FROM debian:buster
+FROM i386/debian:buster
 
 ENV DEBIAN_FRONTEND noninteractive
 
--- scripts/common.org	2019-03-29 18:56:39.152473236 +0100
+++ scripts/common	2019-03-29 18:57:26.020602489 +0100
@@ -94,9 +94,7 @@
 export -f on_chroot
 
 update_issue() {
-	local GIT_HASH
-	GIT_HASH=$(git rev-parse HEAD)
-	echo -e "Raspberry Pi reference ${IMG_DATE}\nGenerated using ${PI_GEN}, ${PI_GEN_REPO}, ${GIT_HASH}, ${1}" > "${ROOTFS_DIR}/etc/rpi-issue"
+        echo -e "DJO Infobord" > "${ROOTFS_DIR}/etc/rpi-issue"
 }
 export -f update_issue
 
